monkey/monkey_island/cc/models/zero_trust/aggregate_finding.py

```diff
@@ -12,15 +12,15 @@ def create_or_add_to_existing(test, status, events):
         :raises: Assertion error if this is used when there's more then one finding which fits the query - this is not
         when this function should be used.
         """
-        existing_findings = Finding.objects(test=test, status=status)
+        existing_findings = Finding.objects(test=test, status=status).exclude('events')
```

Comment:
Can we create a method in Finding/override the regular "objects" method so that it excludes events by default? Do you think that's a good idea? 
Kills all the code dup of writing `.exclude('events')` from all over and prevents new `Finding.objects` calls from doing the same problem

Reply:
Very nice idea!

Reply:
I've looked into it, but seems hard to achieve and not very reliable. objects is not a simple method, it's a queryset object. When you're going into metaclasses, overriding internal objects and other internals it's easy to make mistakes and easy to make code break on package update. I think we can live with this for the time being

Reply:
OK